packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -147,28 +154,29 @@ contract L2Genesis is Deployer {
         }
         vm.stopPrank();
 
-        // Genesis is "complete" at this point, but some hardfork activation steps remain.
-        // Depending on the "Output Mode" we perform the activations and output the necessary state dumps.
-        if (_mode == OutputMode.LOCAL_DELTA) {
-            return;
-        }
-        if (_mode == OutputMode.OUTPUT_ALL) {
+        if (_mode == OutputMode.ALL || _fork == Fork.DELTA && _mode == OutputMode.LATEST) {
```

Comment:
Could we make the logic inside of the `if` expression an internal function that accepts a `Fork` and returns a `bool`? Then its really simple to add new forks

Reply:
done, encapsulated in `writeForkGenesisAllocs`